pkgs/data/themes/plata/default.nix

```diff
@@ -38,8 +40,9 @@ stdenv.mkDerivation rec {
     sassc
     inkscape
     libxml2
-    gnome2.glib.dev
+    glib.dev
   ]
+  ++ stdenv.lib.optionals mateSupport [ gtk3 marco ]
   ++ stdenv.lib.optional telegramSupport zip;
 
   buildInputs = [
```

Comment:
Why does a package that just produces images and gresources containing images have `buildInputs`?

Reply:
I'm not sure; this was copied from `adapta-gtk-theme` which this descends from. I assume the theme has a runtime dependency on `gtk-engine-murrine` (at least for gtk2), and it does specify such in its gtk-2.0 rc files.

Reply:
@worldofpeace Is there anything I need to address here? This line isn't part of the PR.